qa/src/test/java/org/eclipse/kapua/service/datastore/steps/DataStoreServiceSteps.java

```diff
@@ -212,6 +321,17 @@ public void insertRandomMessageIntoDatastore(String msgKey, String idKey) throws
         waitABit(5000);
```

Comment:
Hi Andrej, there are few waitABit() calls in the tests. Could you remove also all these wait call replacing them with the datastore synch call? (by calling refreshAllIndexes in the MessageStoreFacade)